lib/features/debug/view/widget_list/toast_message_view.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+
+class ToastMessageView extends StatelessWidget {
+
+  @override
+  Widget build(BuildContext context) {
+    return const Center(
+      child: Text("Show all toast messages")
```

Comment:
Is this future work? If it is, let's remove it for now, and include in future PR

Reply:
yes it is for future work, I should have leave a comment there. 
Will remove it next commit

Reply:
Please remove in this PR, and add again in related PR